bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -5840,6 +5814,28 @@ void ReplicaImp::tryToGoToNextView() {
 
 IncomingMsgsStorage &ReplicaImp::getIncomingMsgsStorage() { return *msgsCommunicator_->getIncomingMsgsStorage(); }
 
+void ReplicaImp::startSlowPath(SeqNumInfo &seqNumInfo) {
+  seqNumInfo.startSlowPath();
+  metric_total_slowPath_++;
+}
+void ReplicaImp::updateCommitMetrics(const CommitPath &commitPath) {
+  LOG_INFO(CNSUS, "Updating metric" << commitPath);
```

Comment:
I'm not sure there needs to be an INFO level log every time you update a metric. Is this necessary?

Reply:
Removed.